showcase.html

```diff
@@ -487,6 +487,13 @@ <h6>AniLookup</h6>
                 <h6>Kaizen Anime News</h6>
                 <p>Website for updated Anime Information powered by Jikan API</p>
             </div>
+        </a>
+         <a class="item" href="https://anime-cards-alpha.vercel.app/">
+            <img class="lazy" data-src="https://imgur.com/a/RdJuZhO" data-srcset="https://imgur.com/a/RdJuZhO" src="https://imgur.com/a/RdJuZhO" alt="Anime Cards" />
```

Comment:
This is an album link so it wouldn't work as an image source.

1. You'd have to update this to `https://i.imgur.com/za1tQDS.png`
2. Leave src empty as `src=""` since the image will be lazy loaded

Reply:
I fixed it, so sorry for the continuous problems

Reply:
Looks good now, thanks for resolving them